tests/modules/programs/atuin/no-shell.nix

```diff
@@ -19,5 +20,6 @@
   nmt.script = ''
     assertFileNotRegex home-files/.zshrc '@atuin@ init zsh'
     assertFileNotRegex home-files/.bashrc '@atuin@ init bash'
+    assertFileNotRegex home-files/.config/fish/config.fish '@atuin@ init fish'
```

Comment:
The checks here seem invalid. Could you also fix them?

I think they should be rewritten as: `atuin init ...`.
You can check that the tests are valid by turning on `enable${SHELL}Integration` and see them fail.

Reply:
Alright, I've just fixed the tests, but for some reason, enabling the variable doesn't work. Are there any things that I've messed up in the code that would make it not work?